drake/examples/HolonomicDrive/oneWheelGradients.m

```diff
@@ -0,0 +1,43 @@
+% Autogenerated file, from makeGrads.m
```

Comment:
Do we not currently have various `dynamicsGradients.m` checked into git? Or does that get generated on first run?